plugins/lookup/aws_secret.py

```diff
@@ -35,7 +39,8 @@
     description:
         - Join two or more entries to form an extended secret.
         - This is useful for overcoming the 4096 character limit imposed by AWS.
-    type: boolean
+        - No effect when used with I(bypath)
+    type: bool
```

Comment:
Plugins can also use the long form, so it would make sense to not change it here.

Reply:
Reverted this to `boolean`